docs/DeveloperGuide.md

```diff
@@ -200,9 +200,9 @@ The delete student feature is implemented in the `DeleteCommand` class and facil
 
 Given below is an example usage scenario and how the delete student mechanism behaves at each step.
 
-Step 1: The user executes `deleteStud A1234567X` to add a student. The `AddressBookParser` class determines that the command called is `deleteStud`, and therefore creates a new `DeleteCommandParser` instance to parse the command.
+Step 1: The user executes `delete A1234567X` to add a student. The `StudentBookParser` class determines that the command called is `delete`, and therefore creates a new `DeleteCommandParser` instance to parse the command.
```

Comment:
Should it be "to delete a student" instead of "to add a student"?

Reply:
Yes. Thank you so much for spotting this huge oversight!